Doc/whatsnew/3.11.rst

```diff
@@ -919,8 +919,8 @@ Porting to Python 3.11
       #if PY_VERSION_HEX < 0x030900B1
       static inline PyCodeObject* PyFrame_GetCode(PyFrameObject *frame)
       {
-          Py_INCREF(frame->f_code);
-          return frame->f_code;
+          Py_INCREF(frame->code);
+          return frame->code;
```

Comment:
This snippet still isn't right for the new code structure (but it's also already wrong on the main branch)

Reply:
The snippet was actually correct, it's the snippet for 3.8-and-earlier code where `PyFrame_GetCode` didn't exist yet.